### DISCUSSION TITLE
MOES TRV settings (radiators not water )

### DISCUSSION THREAD

**Discussion by raphthibert** [jmcollin78/versatile_thermostat#390] [General]
Hello, 

  First, many thanks to @jmcollin78 for the development! It is very complete and still improving, amazing!
  
  I started using Versatile Thermostat just yesterday and I start to see some settings needed for better temperature control (different insulation of rooms and floors).
  Basically, it is too long to get the set T° on one floor but too quick on another (and exceeding by 0.5 degrees).
  
My configuration is over climate.

  I read all the configuration and understand all the parameters but some things are not clear to me :
     - in central configuration I have TPI settings (coeff int et coefficient ext) : are those parameters active in my case?
     - in each entity, I can activate auto-regulation mode : is this active in my case? working together with the above setting?
     - any other settings to look at?
     
     Thank you for the help.
     Raphaël

**Comment by jmcollin78** [comment]
Hello @raphthibert ,

>  in central configuration I have TPI settings (coeff int et coefficient ext) : are those parameters active in my case?

Not for `over_climate` configuration. In the README you have all the attributes listed and for which configuration it applies,

> in each entity, I can activate auto-regulation mode : is this active in my case? working together with the above setting?

Yes for `over_climate` you have access to the self-regulation (and not the coefs listed above).
